src/Movies.js

```diff
@@ -45,18 +46,19 @@ function renderMovieCount($output, movieCount){
 }
 
 function renderMovie($output, movie) {
-  let html=`<a href="${ urlJoin(baseUrl, movie.url) }" target="_blank" class="movie list-group-item panel ">
-      <div class="title"><span>${movie.title}</span></div> 
+  let html = `
+    <a href="${ urlJoin(baseUrl, movie.url) }" target="_blank" class="movie list-group-item panel ">
+      <div class="title"><span>${movie.title}</span></div>
       <img class="movie-img-small" src="${movie.imgUrl}" data-toggle="modal" data-target=".img-modal">
-      <span class = "content-right">
+      <span class="content-right">
         <ul class="movie-details">
-          <li class="rating">${movie.rating || "not rated" }<span class="icon-star"></span></li> 
-          <li class="metascore">${movie.metascore || "not rated" }<span><img src="../assets/img/meta.png"></span></li>
-          <li class="runtime">${movie.runtime || "N/A"}<span> min</span></li>
+          <li class="rating">${movie.rating || 'not rated'}<span class="icon-star"></span></li>
+          <li class="metascore">${movie.metascore || 'not rated'}<span><img src="../assets/img/meta.png"></span></li>
+          <li class="runtime">${movie.runtime || 'N/A'}<span> min</span></li>
           <li class="genre">${movie.genre}</li>
         </ul>
-          <div class="desc">${movie.desc}</div>
-          <div class="votes">Votes: ${movie.votes || "N/A"}</div>
+        <div class="desc">${movie.desc}</div>
+        <div class="votes">Votes: ${movie.votes || 'N/A'}</div>
```

Comment:
I agree with single quotes, when I think about it, those are actually JS lines of code